### PR TITLE
Meals can now be created by caterings

### PR DESCRIPTION
**Pull request by pgorni**
Also changed the routes according to that.

### CODE REVIEW

File: app/controllers/meals_controller.rb [hunk 1]
```diff
@@ -15,8 +15,27 @@ def show
...
+  end
+
+  def create
+    @meal = Meal.new(
+      name: params.require(:meal).permit(:name),
+      catering: current_catering
+    )
+
+    if @meal.save
+      flash[:success] = "Meal successfully created!"
```
Comment (kompnet) [inline app/controllers/meals_controller.rb]: Use :notice instead of :success

File: app/controllers/meals_controller.rb [hunk 2]
```diff
@@ -15,8 +15,27 @@ def show
...
+      redirect_to new_catering_meal_path
+    else
+      flash[:error] = "Something failed."
```
Comment (kompnet) [inline app/controllers/meals_controller.rb]: Use :alert instead of :error

**Comment by returnv01d** [comment]
Its ok :)

File: db/schema.rb [hunk 3]
```diff
@@ -68,7 +64,7 @@
   end
 
   create_table "ingredients", force: :cascade do |t|
-    t.string "name", null: false, unique: true
+    t.string "name", null: false
```
Comment (Lemcur) [inline db/schema.rb]: bump, check what happened

**Comment by returnv01d** [comment]
Its' ok :+1:

**Pull request merged**